### PR TITLE
Post processing pressure power and viscous dissipation

### PR DESCRIPTION
**Pull request by blaisb**
# Description of the problem

- For the analysis of the turbulent taylor-couette problem we need to be able to postprocess the power of the pressure (\nabla p \cdot u) and the viscous dissipation (\tau \cdot \nabla u) in order to do the full kinetic energy balance

# Description of the solution

- Added both features as well as their tables and serialization
- Modified the application test for taylor couette to include the viscous dissipation. If you multiply the viscous dissipation by the volume of the triangulation you get exactly the torque * the angular velocity, which is exactly what we expect
- Added the documentation of the features.

# Comments

- I took the opportunity to make some more things const that should have been const

### CODE REVIEW

**Review by lpsaavedra — CHANGES_REQUESTED** [review]
Nice improvements... just a few small comments from my side

File: source/solvers/postprocessing_cfd.cc
Comment (PierreLaurentinCS) [inline source/solvers/postprocessing_cfd.cc:487]: Why the parenthesis around the returned variable? Is it for aesthetic?

File: source/solvers/postprocessing_cfd.cc
Comment (PierreLaurentinCS) [inline source/solvers/postprocessing_cfd.cc:599]: Why the parenthesis around the returned variable? Is it for aesthetic?

**Review by PierreLaurentinCS — CHANGES_REQUESTED** [review]
Very good job, I just want to make sure there is no confusion between power and work of pressure

File: include/core/parameters.h
Comment (blaisb) [reply include/core/parameters.h:822]: Yes it's power everywhere, power is the right terminology I made the mistake at first

**Review by lpsaavedra — CHANGES_REQUESTED** [review]
Some additional minor comments

**Review by lpsaavedra — APPROVED** [review]

**Comment by blaisb** [comment]
@PierreLaurentinCS I need your approval before I cna merge

**Review by PierreLaurentinCS — APPROVED** [review]
Everything is good for me! Good job :thumbsup: :thumbsup: :thumbsup:

**Pull request merged**
